SPECS/runc/runc.spec

```diff
@@ -2,8 +2,8 @@
 Summary:        CLI tool for spawning and running containers per OCI spec.
 Name:           runc
 # update "commit_hash" above when upgrading version
-Version:        1.1.12
```

Comment:
Have we reviewed the changelog to see what's changed between 1.1.12 and 1.2.1? Were there breaking changes that would affect containerd v1 and/or other container hosting scenarios? What other risks might there be of this upgrade?

Also, I see that runc *1.2.2* has already been released and claims to have fixes for some regressions introduced in 1.2.0. Should we be picking that up instead?

Reply:
I see a number of "breaking changes" listed in the 1.2.0 series in the release notes for runc, but don't have the right domain knowledge to know how likely they are to be impactful to any of our customers. Is this something that's been previously evaluated?

Reply:
@reubeno Some of the breaking changes are validated by running the existing conformance tests, performance tests, pre-release tests and AKS e2e tests against both containerd 1.7.13+runc 1.2.1 and containerd 2.0.0+runc 1.2.1, and no regression was found. (e.g. azl 3.0 AKS image and the associated tests: https://dev.azure.com/mariner-org/mariner/_build/results?buildId=673688&view=results). Although we don't have dedicated containerd and runc tests yet, the existing tests (especially AKS e2e) include the most common scenarios that perform container operations in k8s environment. We also provided the upgraded rpms to COSMIC team who is going to run their workload for validation.

runc 1.2.1 is required by containerd 2.0.0 ([ref](https://github.com/containerd/containerd/blob/v2.0.0/script/setup/runc-version)), and it would be better upgrade to the latest 1.2.2 version to include more fixes and align with the [Fedora offerings](https://src.fedoraproject.org/rpms/runc). I will update this PR with the runc 1.2.2.